src/lib/Theme.ts

```diff
@@ -365,28 +366,25 @@ export default class Theme {
                 shouldDelete = false;
                 throw new CommandError(`Folder ${options.name} already exists`);
             }
-            const themeType = await Theme.selectThemeType();
```

Comment:
Why selectThemeType function is not removed?

Reply:
Removed